### ISSUE TITLE
[BUG] H025 false positive </a>

### ISSUE THREAD

**Issue by jayvdb** [Riverside-Healthcare/djLint#83]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## System Info
 - OS: Windows
 - Python Version: 3.7
 - djLint Version: 0.5.5

## Issue
The following html causes `H025 Tag seems to be an orphan. </a>`

```html
{% block back_link %}
  <li class="no-padding">
    <a class="collapsible-header" href="/admin">
      <i class="material-icons">home</i>Admin Home
    </a>
  </li>
{% endblock %}
```

## How To Reproduce
see above
